CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### Bug fixes
```

Comment:
You'll have to fix this one as well. :-)

Reply:
I'm terrible at these. :flushed: